shared/src/commonMain/kotlin/di/Helper.kt

```diff
@@ -3,9 +3,16 @@ package di
 import org.koin.core.context.startKoin
 import org.koin.dsl.KoinAppDeclaration
 
+private val allModules = listOf(modules, clientsModule, servicesModule, repositoriesModule, platformModules())
```

Comment:
I do not see the added value of extracting this to another value 😅

Reply:
The way to initialize koin is also slightly different for android and ios so we extracted it into a variable so we don't have to type it twice.

Reply:
Okay nice!